pkg/kubemark/hollow_proxy.go

```diff
@@ -42,6 +45,24 @@ type HollowProxy struct {
 	ProxyServer *proxyapp.ProxyServer
 }
 
+type FakeProxyHandler struct{}
```

Comment:
The reason why we need those typecasts is because `proxier` is defined as a variable of type `interface{}`.
And the reason for defining it as an interface is so that I can use a common variable for taking value based on the case (i.e. FakeProxier or proxy.ProxyProvider).
I can combine FakeProxyHandler and FakeProxier, but IIUC I still need the typecasting (as otherwise it complains `cannot use proxier (type interface {}) as type "k8s.io/kubernetes/pkg/proxy/config".ServiceHandler in field value`)